website/content/en/preview/contributing/development-guide.md

```diff
@@ -21,42 +21,64 @@ The following tools are required for contributing to the Karpenter project.
 
 ### Setup / Teardown
 
-Based on how you are running your Kubernetes cluster, follow the [Environment specific setup](#environment-specific-setup) to configure your environment before you continue. You can choose to either run the Karpenter controller locally on your machine, pointing to the Kubernetes cluster specified in your `~/.kube/config` or inside the Kubernetes cluster specified in your `~/.kube/config` deployed with [Helm](https://helm.sh/).
+#### Container Repository
```

Comment:
I don't know that you need to go through the whole `git clone` thing, but I think a short introductory section to begin the page would be useful. At least mention the location of the Karpenter repo and add a general description of what you are doing in this document. For example, mention that you need to add Karpenter components to a container registry, create a cluster, and so on to be able to build and run Karpenter.